### PR TITLE
PR 57

### PR DESCRIPTION
**Pull request by ericsorenson**
- [7387 Order and Opinion - Coming Soon](https://github.com/flexion/ef-cms/issues/7387) 
- [7378 BUG: Migrated PSDE is not displaying "Sign Proposed Stipulated Decision" option ](https://github.com/flexion/ef-cms/issues/7378) 
- [7375 BUG: Add Court Issued Docket Entry - "Judge" title not added to docket entry ](https://github.com/flexion/ef-cms/issues/7375) 
- [7358 Search Usability: Load more, and retain results](https://github.com/flexion/ef-cms/issues/7358) 
- [7355 Exact matches: Case Search Usability](https://github.com/flexion/ef-cms/issues/7355) 
- [7329 BUG: Clearing Pending Item does not remove Block on case](https://github.com/flexion/ef-cms/issues/7329) 
- [7321 BUG: Legacy Petitioner: Error received when editing petitioner contact information ](https://github.com/flexion/ef-cms/issues/7321)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -334,16 +334,16 @@ petitionsclerk@example.com
...
 reportersOffice@example.com
 trialclerk@example.com
-judge.armen@example.com
-colvinsChambers@example.com
 judge.ashford@example.com
 ashfordsChambers@example.com
 judge.buch@example.com
 buchsChambers@example.com
-judge.carluzzo@example.com
+stjudge.carluzzo@example.com
```
Comment (julialeague) [inline README.md:341]: is this "st" intentional?

Comment (rachaelparris) [reply README.md:341]: Yes, it's for `special trial judge`. This is the email we use when creating the judge users in `judge_users.csv`, so we just updated this one to match.

Comment (mmarcotte) [reply README.md:341]: hm, but these users have already been created in the system and are being used. I guess I need to figure out what action needs to be done Court side for this, if any?

File: web-client/integration-tests-public/unauthedUserSeesStrickenDocketEntry.test.js [hunk 2]
```diff
@@ -86,7 +86,8 @@ describe('Unauthed user views stricken docket entry for order', () => {
   unauthedUserAttemptsToViewStrickenDocumentUnsuccessfully(test);
 });
 
-describe('Unauthed user searches for stricken order', () => {
+// Temporarily disabled for story 7387
+describe.skip('Unauthed user searches for stricken order', () => {
```
Comment (julialeague) [inline web-client/integration-tests-public/unauthedUserSeesStrickenDocketEntry.test.js:90]: I just want to check that there's a process set up for unskipping these once the order and opinion searches are reenabled - would that be an AC on an existing issue, or making sure that another issue is logged to unskip these?

File: README.md (hunk 1 above)
Comment (mmarcotte) [reply README.md:341]: Thanks @rachaelparris! Yeah, @julialeague , the one we have loaded in the system (and are using) is "stjudge.carluzzo@example.com". 👍

**Comment by julialeague** [comment]
- [x] review the pull request itself, to get oriented
	- [x] read the description of the pull request, which should summarize the changes made
	- [x] read through every task on the Scrum board that's encompassed by this pull request
	- [x] read the description of the commits that comprise the pull request
- [x] If changes were made in the UI (if not, skip):
  - [x] If desired, run `docker system prune` to remove any unused docker images from previous code review.
  - [x] Fetch the pull request for the sprint (e.g., `git fetch origin pull/{PR #}/head:sprint-{Sprint #}`), and then switch to that branch (e.g. `git checkout sprint-{Sprint #}`)
  - [x] stand up the site locally, with `./docker-run.sh`
	- [ ] test all functionality in all major browsers, emphasizing the functionality that this pull request addresses
	- [ ] for internal Court functionality, perform the most thorough testing in Chrome, though also test in Edge and Firefox
	- [ ] for public-facing functionality, test in browsers consistent with [public browser use data](https://analytics.usa.gov/)
	- [ ] test in Mobile Safari and Mobile Chrome (or an emulator like Chrome DevTools), with the caveat that not all internal Court functionality will be necessary on these platforms
	- [x] use an automated audit tool for code quality and practices (recommended: [Chrome DevTools](https://developers.google.com/web/tools/chrome-devtools/), aka [Lighthouse](https://chrome.google.com/webstore/detail/lighthouse/blipmdconlkpinefehnmjammfjpmpbjk))
		- [x] look at efficiency of page loads, asset sizes, HTTP connection management, etc.
	- [x] review for accessibility
		- [x] use an automated audit tool, such as Chrome Audit or aXe
		- [x] navigate site only with the keyboard
		- [x] use VoiceOver or Narrator to navigate the site with audio only, with the display turned off
		- [x] manually test anything that pa11y cannot test automatically (e.g., contrast of text over images)
- [x] [review static code analysis results](https://sonarcloud.io/organizations/flexion-github/projects) in the vendor’s SonarCloud environment
- [x] examine OWASP ZAP output in `docs/`, to ensure that any errors are known to be false positives or have been previously declared to be acceptable
- [x] skim all new code, in the context of existing code, [looking for problems](#what-we-look-for) (knowing that the vast majority of new code will be covered by tests)
- [x] review all tests
	- [x] look at code coverage of tests in [the vendor's SonarCloud environment](https://sonarcloud.io/organizations/flexion-github/projects)
	- [x] methodically review all new tests for correctness, quality of naming
- [x] determine what code isn’t tested, review that rigorously
- [x] review documentation to ensure that it matches changes
	- [x] user-visible changes (including API users like the IRS) are documented in CHANGELOG.md (which is linked from US Tax Court website).
- [x] provide comments on the pull request on GitHub, as necessary
	- [x] for comments that are specific to a particular line of code, comment on those specific lines
- [ ] for each feature-level bug (i.e., it’s working as designed, but designed wrong), open a new issue and put it in the backlog

File: web-client/integration-tests-public/unauthedUserSeesStrickenDocketEntry.test.js (hunk 2 above)
Comment (kkoskelin) [reply web-client/integration-tests-public/unauthedUserSeesStrickenDocketEntry.test.js:90]: Comment and link added in AC at https://github.com/flexion/ef-cms/issues/7368 and https://github.com/flexion/ef-cms/issues/7376

**Review by mmarcotte — APPROVED** [review]

**Pull request merged**